xml/net_bonding.xml

```diff
@@ -254,11 +256,11 @@ ifcfg-eth1
   <para>
    <literal>BOOTPROTO=none</literal> uses the <command>ethtool</command>
    options (when provided), but does not set the link up on <command>ifup
-   eth0</command>. The reason is that the slave interface is controlled by the
-   bond master.
+   eth0</command>. The reason is that the bond port interface is controlled by the
+   bond parent.
```

Comment:
I personally don't like ```parent``` replacement for the master device. I've also used that as last instance only. So, I'd most probably use ```bond device``` here.